src/Makefile.am

```diff
@@ -497,6 +497,10 @@ if TARGET_WINDOWS
 bitcoind_SOURCES += bitcoind-res.rc
 endif
 
+# LIBBITCOIN_SERVER added twice intentionally. Our libs are static and have circular
+# dependencies, Linux linkers which process libraries in command line order and only
```

Comment:
The second sentence here isn't grammatical and is hard to understand (I'm not even sure it makes sense). Would replace it with a shorter comment and a link.

```shell
# Libraries below may be listed more than once to resolve circular dependencies (see
# https://eli.thegreenplace.net/2013/07/09/library-order-in-static-linking#circular-dependency)
```